src/content/docs/accounts/accounts-billing/account-setup/multiple-logins-found.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "Login message: Multiple logins found"
+tags:
+  - Accounts
+  - Accounts and billing
+  - Account setup
+metaDescription: "When logging into New Relic, you may see a 'multiple logins found' message and be asked to verify your email."
+redirects:
+---
+
+import loginMultipleAccountsView from 'images/login-multiple-accounts-view.png'
+
+If your email is associated with multiple user records, you'll be given a `Multiple logins found` message and be asked to verify your email. This can happen due to your having access to multiple New Relic organizations, or to multiple [authentication domains](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more) in an organization. 
```

Comment:
```suggestion
If your email is associated with multiple user records, you'll see a `Multiple logins found` message at login, then you'll be asked to verify your email. This might happen if you have access to multiple New Relic organizations, or to multiple [authentication domains](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more) in an organization. 
```

Reply:
Just to simplify/smooth out the beginning sentences